client/src/components/Home/Home.js

```diff
@@ -37,64 +34,22 @@ class Home extends React.Component {
               searchData={searchData}
             />
           </div>
-
+          {/* W.8.23.20 QUESTION: should we mark this as nav? It's essentially a secondary nav menu, so I wasn't sure. */}
```

Comment:
I think a nav here would be appropriate here.  According the to MDN docs multiple navs are fine to promote accessibility: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav